test/suite/unit/Exception/StaleUpdateException.spec.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+declare(strict_types=1); // @codeCoverageIgnore
+
+namespace Rinq\Exception;
+
+use Rinq\Ident\PeerId;
+use Rinq\Ident\Reference;
+use Rinq\Ident\SessionId;
+
+describe('StaleUpdateException', function () {
+    beforeEach(function () {
+        $this->reference = Reference::create(SessionId::create(PeerId::create(111, 222), 333), 444);
+        $this->subject = new StaleUpdateException($this->reference);
+    });
+
+    describe('->construct()', function () {
+        it('should construct a meaningful exception message', function () {
+            expect($this->subject->getMessage())->to->equal(
+                'Can not update or close 6F-00DE.333@444, the session has ' .
+                'been modified since that revision.'
```

Comment:
I generally only split the strings that exceed 120 chars. I have indented the second line in those cases.